api/query/cache/index/index_filter_test.go

```diff
@@ -13,9 +13,9 @@ import (
 	mapset "github.com/deckarep/golang-set"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
-	metrics "github.com/web-platform-tests/wpt.fyi/shared/metrics"
 	"github.com/web-platform-tests/wpt.fyi/api/query"
 	"github.com/web-platform-tests/wpt.fyi/shared"
+	metrics "github.com/web-platform-tests/wpt.fyi/shared/metrics"
```

Comment:
Nit:
```suggestion
	"github.com/web-platform-tests/wpt.fyi/shared/metrics"
```
Again `gofmt` would catch this.

Reply:
umm I tried to go fmt on this file and didn't see the suggested change, plus `metrics` was there originally